app/models/case_request.rb

```diff
@@ -5,10 +5,10 @@ class CaseRequest
     :confirmation_code,
     :fees
 
-  def initialize(opts = {})
-    @case_reference = opts[:case_reference]
-    @confirmation_code = opts[:confirmation_code]
-    @fees = []
+  def initialize(case_reference, confirmation_code)
+    @case_reference    = case_reference
+    @confirmation_code = confirmation_code
+    @fees              = []
```

Comment:
I quite like it, although admittedly it's hard to keep consistent. I have an editor extension that does it for me on a keystroke though.